src/main/java/org/acoli/conll/rdf/CoNLLRDFUpdaterFactory.java

```diff
@@ -0,0 +1,105 @@
+				"read TTL from stdin => update CoNLL-RDF", new Option[] {
+						// Define cli options in the correct order for the help-message
+						Option.builder("loglevel").hasArg().desc("set log level to LEVEL").argName("level").build(),
+						Option.builder("threads").hasArg()
+								.desc("use T threads max\ndefault: half of available logical processor cores")
+								.type(Number.class).build(),
+						Option.builder("lookahead").hasArg().desc("cache N further sentences in lookahead graph")
+								.type(Number.class).build(),
+						Option.builder("lookback").hasArg().desc("cache N preceeding sentences in lookback graph")
+								.type(Number.class).build(),
+						new Option("prefixDeduplication", false, "Remove duplicates of TTL-Prefixes"),
+						Option.builder("custom").hasArg(false).desc("use custom update scripts")
+								./* required(). */build(),
+						Option.builder("model").hasArgs().desc("to load additional Models into local graph").build(),
+						Option.builder("graphsout").hasArgs().desc(
+								"output directory for the .dot graph files\nfollowed by the IDs of the sentences to be visualized\ndefault: first sentence only")
+								.build(),
+						Option.builder("triplesout").hasArgs()
+								.desc("same as graphsout but write N-TRIPLES for text debug instead.").build(),
+						Option.builder("updates").hasArgs()
+								.desc("followed by SPARQL scripts paired with {iterations/u}").build() },
+				CoNLLRDFUpdater.LOG).parseArgs(args);
+
+		if (cmd.hasOption("threads")) {
+			updater.setThreads(((Number) cmd.getParsedOptionValue("threads")).intValue());
+		}
+		if (cmd.hasOption("lookahead")) {
+			updater.activateLookahead(((Number) cmd.getParsedOptionValue("lookahead")).intValue());
+		}
+		if (cmd.hasOption("lookback")) {
+			updater.activateLookback(((Number) cmd.getParsedOptionValue("lookback")).intValue());
+		}
+		if (cmd.hasOption("prefixDeduplication")) {
+			updater.activatePrefixDeduplication();
+		}
+		// READ GRAPHSOUT PARAMETERS
+		if (cmd.hasOption("graphsout")) {
+			String[] graphsoutArgs = cmd.getOptionValues("graphsout");
+			String outputDir = graphsoutArgs[0];
+			List<String> outputSentences = Arrays.asList(Arrays.copyOfRange(graphsoutArgs, 1, graphsoutArgs.length));
+			updater.activateGraphsOut(outputDir, outputSentences);
+		}
+		// READ TRIPLESOUT PARAMETERS
+		if (cmd.hasOption("triplesout")) {
+			String[] triplesoutArgs = cmd.getOptionValues("triplesout");
+			String outputDir = triplesoutArgs[0];
+			List<String> outputSentences = Arrays.asList(Arrays.copyOfRange(triplesoutArgs, 1, triplesoutArgs.length));
+			updater.activateTriplesOut(outputDir, outputSentences);
+		}
+
+		if (cmd.hasOption("model")) {
+			for (Option opt : cmd.getOptions()) {
+				if (opt.getOpt() == "model") { // opt.equals(model)
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [ES: Comparison of String objects using == or != (ES_COMPARING_STRINGS_WITH_EQ)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq).
This code compares java.lang.String objects for reference equality using the == or != operators. Unless both strings are either constants in a source file, or have been interned using the String.intern() method, the same string value may be represented by two different String objects. Consider using the equals(Object) method instead.